package.json

```diff
@@ -89,12 +86,12 @@
   },
   "scripts": {
     "build:evergreen": "node .evergreen/generate_evergreen_tasks.js",
-    "build:ts": "gulp compile",
-    "build:dts": "gulp definition",
-    "build:docs": "gulp doc",
+    "build:ts": "rimraf lib && tsc",
+    "build:dts": "npm run build:ts && api-extractor run --typescript-compiler-folder node_modules/typescript --local && find lib -name '*.d.ts*' -exec sh -c 'rimraf \"$0\"' {} \\;",
```

Comment:
Yea you are right, I used find here instead of just globing because it wasn't deleting all the .d.ts files... turns out my shell (fish) responds to the double star `lib/**/*.d.ts` differently... anyway incoming fix, just pass the glob through to rimraf and it will use node's expansion instead of the local shells

That being said, should I pull in a module that runs a series of npm scripts? because `&&` also won't work on windows

Reply:
Something like this: https://www.npmjs.com/package/npm-run-all